src/map/itemdb.hpp

```diff
@@ -3015,9 +3009,11 @@ enum e_delay_consume : uint8 {
 };
 
 /// Enum for different ways to search an item group
-enum e_group_search_type : uint8 {
-	GROUP_SEARCH_BOX = 0, // Always return an item from the group, rate determines which item is more likely to be returned
-	GROUP_SEARCH_DROP = 1, // Pick one item from the group and check use rate as drop rate, on fail, do not return any item
+enum e_group_draws_type : uint8 {
```

Comment:
Changing the enum now is problematic, you will never get it merged with the main PR. Please just add your changes to the main PR then.

But also I don't understand this description at all...